src/mpid/ch4/src/ch4_send.h

```diff
@@ -46,7 +46,7 @@ MPL_STATIC_INLINE_PREFIX int MPID_Send(const void *buf,
 #ifndef MPIDI_CH4_EXCLUSIVE_SHM
```

Comment:
It will most likely go away from that code and abstracted in a different way, probably inside a dispatch macro.